src/editor/object_option.hpp

```diff
@@ -443,6 +443,21 @@ class ButtonOption : public ObjectOption
   ButtonOption& operator=(const ButtonOption&) = delete;
 };
 
+class StringArrayOption : public ObjectOption
+{
+public:
+  StringArrayOption(const std::string& text, const std::string& key, std::vector<std::string>* items);
+
+  virtual void save(Writer& write) const override;
+  virtual std::string to_string() const override { return {}; }
```

Comment:
Why an empty string there?

It's used at least for displaying on tooltips, it would be preferable to return something if possible.